runtime/lua/vim/lsp/completion.lua

```diff
@@ -676,14 +682,15 @@ end
 --- @class vim.lsp.completion.BufferOpts
 --- @field autotrigger? boolean  Default: false When true, completion triggers automatically based on the server's `triggerCharacters`.
 --- @field convert? fun(item: lsp.CompletionItem): table Transforms an LSP CompletionItem to |complete-items|.
+--- @field fuzzyopts? table vim.fn.matchfuzzy.dict
```

Comment:
Just above this is a `convert` function. That's a big hint that a `filter` function would be a natural counterpart.

This pattern keeps coming up. I hope that future PRs will consider this pattern, instead of adding random options.

Reply:
There's https://github.com/neovim/neovim/pull/30936 for that. Looks like we don't need fuzzyopts

Reply:
I can't guess how to pass options to matchfuzzy called in #30936.

Reply:
#30936 (outdated at the moment) has `filter` callback that could be used to call `matchfuzzy` with custom options instead. `filter` doesn't seem to be present on the `master` branch.